apps/web/src/components/organization-select.tsx

```diff
@@ -85,28 +86,31 @@ export default function OrganizationSelect() {
   });
 
   const { mutate: createOrganizationMutation } = useMutation<
-    InsertOrganization | undefined,
+    Organization | null,
     Error,
     InsertOrganization
   >({
     mutationKey: ["create", "organization"],
-    mutationFn: async (insertData) => createOrganization(insertData),
-    onSuccess: (data) => {
-      if (!!data) {
+    mutationFn: async (insertData) => {
+      const [newOrg, err] = await createOrganization(insertData);
+
+      if (err !== null) {
+        toast({
+          title: "Error",
+          description: err,
+        });
+      } else {
         toast({
           title: "Success",
           description: "Organization created successfully",
         });
-        router.push(`/dashboard/${data.name}/overview`);
+
+        router.push(`/dashboard/${newOrg.name}/overview`);
         setIsModalOpen(false);
         refetch();
       }
-    },
```

Comment:
why delete this toast?

Reply:
It's been moved to the `mutationFn` since it cannot throw and the error is caught within `createOrtanization`